docker-for-mac/release-notes.md

```diff
@@ -13,10 +13,19 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.5
+2020-03-30
```

Comment:
@mat007 We need to update the release date.

Reply:
Ah yes, doing it!

Reply:
Done.